python-package/requirements.txt

```diff
@@ -1,2 +1,3 @@
 grpcio-tools==1.7.0
-twine==1.9.1
\ No newline at end of file
+twine==1.9.1
+requests==2.21.0
```

Comment:
Should we define `requests` as optional dependency?
It's used only in a single HTTP client and I'm not sure if we want to install requests even when we don't use it.